pyiron_base/jobs/job/util.py

```diff
@@ -117,8 +117,23 @@ def _get_project_for_copy(job, project, new_job_name):
 
 
 def _get_safe_job_name(
-    name: str, ndigits: Union[int, None] = 8, special_symbols: Union[Dict, None] = None
+        name: Union[str, tuple],
+        ndigits: Union[int, None] = 8,
+        special_symbols: Union[dict, None] = None
 ):
+    """
+    Sanitize a job name, optionally appending numeric values.
+
+    Args:
+        name (str|tuple): The name to sanitize, or a tuple of the name and any number
+            of numeric values to append with '_' in between.
+        ndigits (int|None): How many digits to round any floating point values in a
+            `name` tuple to. (Default is 8; to not round at all use None.)
+        special_symbols:
```

Comment:
```suggestion
        special_symbols (dict): Conversions of special symbols to apply. This will be applied to the default conversion dict, which contains: DEFAULT_CONVERSION
```

And `DEFAULT_CONVERSION` should be programmatically replaced by `_special_symbol_replacements`